test/middleware/busted_tracer.rb

```diff
@@ -0,0 +1,26 @@
+require "logger"
```

Comment:
`test/middleware/busted_tracer.rb` is the same as [`lib/busted/middleware/busted_tracer.rb`](https://github.com/HParker/busted/blob/652cfd30e482f0c8fe2abc3fb635a716c6d0aa7a/lib/busted/middleware/busted_tracer.rb). Why must we copy the file into the `test` directory?